common/structs.go

```diff
@@ -24,4 +24,5 @@ type RunArgs struct {
 	PodCidr    string
 	SvcCidr    string
 	Provider   string
+	CustomEnv  []string
```

Comment:
i think the arg `CustomEnv ` should be `map[string]string`  type. why we use `[]string`.